ui/eureka/eureka.js

```diff
@@ -115,19 +115,19 @@ let Options = {
   Regex: {
     en: {
       'gFlagRegex': Regexes.parse(/00:00(?:38:|..:[^:]*:)(.*)Eureka (?:Anemos|Pagos|Pyros|Hydatos) \( (\y{Float})\s*, (\y{Float}) \)(.*$)/),
-      'gTrackerRegex': Regexes.parse(/(?:https:\/\/)?ffxiv-eureka\.com\/(?!maps\/)(\S*)\/?/),
+      'gTrackerRegex': Regexes.parse(/(?:https:\/\/)?ffxiv-eureka\.com\/(?!maps\/|logograms|local-tracker|trackers|BA-visual-guide\.pdf)(\S*)\/?/),
```

Comment:
How long are the tracker links? Looking through my logs, I only see ones that are 6 characters long. 

What about: `Regexes.parse(/(?:https:\/\/)?ffxiv-eureka\.com\/(\S{6})\/?/),` instead?